src/templates/drawer.jsx

```diff
@@ -254,6 +259,7 @@ class Drawer extends React.Component {
         const layeredOffset = 11;
         const scrollPosition = window.pageYOffset;
         const zIndex = 10002; // adding 2 accounts for the frist .drawer and .drawer-dimmers- z-indexes
+        const drawerWidth = mobileMaxWidth ? mobileMaxWidth + 1 : DEFAULT_DRAWER_WIDTH; // This implies if mobileMaxWidth is set and maxWidth not set
```

Comment:
Curious ... why + 1?

Reply:
Like if, `mobileMaxWidth=1018` then drawer width would be around 1018. Previously, it was handled like this for 767.